src/Fixer/LanguageConstruct/CombineConsecutiveIssetsFixer.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class CombineConsecutiveIssetsFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Using `isset(X) &&` multiple times should be done in one call.',
+            [new CodeSample("<?php\n\$a = isset(\$a) && isset(\$b);")]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // should ran before NoMultilineWhitespaceBeforeSemicolonsFixer, NoTrailingWhitespaceFixer and NoWhitespaceInBlankLineFixer.
+        return 1;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAllTokenKindsFound([T_ISSET, T_BOOLEAN_AND]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $tokenCount = $tokens->count();
+
+        for ($index = 1; $index < $tokenCount; ++$index) {
+            if (!$tokens[$index]->isGivenKind(T_ISSET) || $tokens[$tokens->getPrevMeaningfulToken($index)]->equals('!')) {
+                continue;
+            }
+
+            $issetInfo = $this->getIssetInfo($tokens, $index);
+            $issetCloseBraceIndex = end($issetInfo); // ')' token
+            $insertLocation = prev($issetInfo) + 1; // one index after the previous meaningful of ')'
+
+            $andAndTokenIndex = $tokens->getNextMeaningfulToken($issetCloseBraceIndex);
```

Comment:
`andAnd` ?

Reply:
`&&` -> `&` `&` -> `and` `and` -> `andAnd`, no?

Reply:
`&&` is `T_BOOLEAN_AND` token, thus maybe `$booleanAndTokenIndex` ?
just to avoid confusion